### ISSUE TITLE
Can't perform actions: dial unix /run/user/0/keybase/keybased.sock: connect: permission denied

### ISSUE THREAD

**Issue by xubin990510** [keybase/client#3745]
I can't do anything except reading helps. Also /run/user/0 is empty.

My system system CentOS 7 64bit. 

my log id: b9f8ef6bed2d1d371f5fdb1c

**Comment by cjb** [comment]
It sounds like you might be trying to run keybase as root. Is that true? That doesn't work, it needs to be a normal user.

We don't support CentOS yet, so it might not work.

**Comment by xubin990510** [comment]
@cjb 
Nope, I'm not trying to run keybase as root. In fact, if I try to do so, keybase will give me a nice warning about UID 0 is not supported.

My normal user's ID is 1000, if that helps.

Thanks a lot.

**Comment by cjb** [comment]
Okay, better wait for us to work out what's happening on CentOS.

**Comment by xubin990510** [comment]
Thanks!

**Comment by xubin990510** [comment]
@cjb 
I found that if I chmod /run/user/0 to 777 (original: 700) and then run_keybase, things will work fine.

**Comment by Scath-Enfys** [comment]
@xubin990510 I wouldn't recommend that. That gives any app access to root's currently attached sockets.
